plutus-contract/src/Language/Plutus/Contract/Request.hs

```diff
@@ -115,3 +115,7 @@ select = (<|>)
 
 checkpoint :: forall s e a. (Aeson.FromJSON a, Aeson.ToJSON a) => Contract s e a -> Contract s e a
 checkpoint = Contract . CJSONCheckpoint . unContract
+
+-- | Transform any exceptions thrown by the 'Contract' using the given function.
```

Comment:
I was going to say we should call this `mapError` like `Control.Monad.Except`... but apparently they do call it `withError`, bizarrely. I guess this is the "standard" name, then.

Reply:
Hm, maybe we could just add a `Bifunctor` instance?

Reply:
good idea

Reply:
`ExceptT` can't have a `Bifunctor` instance since it's arguments are in the wrong order. Also, we should probably keep the explicit functions for clarity even if we add the instance.

Reply:
Hm, `Bifunctor` competes with `MMorph` for the 2nd type parameter :(

Reply:
Ah, I see, `Resumable` has the wrong shape. Never mind, then.